apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -463,6 +473,21 @@ class SparseNDArrayRead(_SparseNDArrayReadBase):
         Maturing.
     """
 
+    def dask_array(
+        self,
+        **config: Unpack[DaskConfig],
```

Comment:
what is the rationale for using unpacked params here, but using a dict for `to_anndata`?  Should they be consistent for better API UX?

Reply:
On `to_anndata`, the `dask=dict(…)` indirection groups and namespaces the Dask-specific configs (as opposed to e.g. putting them directly on `to_anndata`, with a `dask_` prefix on each kwarg name, and needing to verify consistency about which are present/absent).

When calling `SparseNDArray.dask_array`, it's clearer that the kwargs are Dask-related / that we're executing "Dask mode."